crypto/evp/ec_support.c

```diff
@@ -10,7 +10,7 @@
 #include <string.h>
 #include <openssl/ec.h>
 #include "crypto/ec.h"
-#include "internal/e_os.h" /* strcasecmp required by windows */
+#include "internal/nelem.h"
```

Comment:
Why is this inclusion necessary?

Reply:
https://github.com/openssl/openssl/blob/e915c3f5381cd38ebdc1824c3ba9896ea7160103/crypto/evp/ec_support.c#L125-L127 uses OSSL_NELEM

Reply:
Ah, and `e_os.h` included that, gotcha!